boards/weio/include/periph_conf.h

```diff
@@ -33,15 +33,15 @@ extern "C" {
 #define TIMER_IRQ_PRIO      1
 
 /* Timer 0 configuration */
-#define TIMER_0_DEV         LPC_CT32B0
+#define TIMER_0_DEV         LPC_CT32B1
 #define TIMER_0_CHANNELS    4
 #define TIMER_0_PRESCALER   (48U)
 #define TIMER_0_MAX_VALUE   (0xffffffff)
-#define TIMER_0_CLKEN()     (LPC_SYSCON->SYSAHBCLKCTRL |= (1 << 9))
-#define TIMER_0_CLKDIS()    (LPC_SYSCON->SYSAHBCLKCTRL &= ~(1 << 9))
-#define TIMER_0_ISR         isr_ct32b0
-#define TIMER_0_IRQ         TIMER_32_0_IRQn
-/** @} */
+#define TIMER_0_CLKEN()     (LPC_SYSCON->SYSAHBCLKCTRL |= (1 << 10))
+#define TIMER_0_CLKDIS()    (LPC_SYSCON->SYSAHBCLKCTRL &= ~(1 << 10))
+#define TIMER_0_ISR         isr_ct32b1
+#define TIMER_0_IRQ         TIMER_32_1_IRQn
```

Comment:
These timer changes seem unrelated to PWM. Do you mind opening a separate PR for it? Or keep the PR title more generic.